src/main/java/cz/cvut/kbss/termit/util/Constants.java

```diff
@@ -22,6 +22,11 @@
  */
 public class Constants {
 
+    /**
+     * Fragment of the glossary.
+     */
+    public static final String GLOSSARY_FRAGMENT = "glosář";
```

Comment:
I'm sorry but I don't see this as much of an improvement. This will still require changing and recompiling the application just to, for example, switch to English. Why is this not a configurable property just like term or file separator?

Reply:
I get this, but TBH my intention was just to have it marked somewhere. It is a part of the IRI, not label, so I do not see much value in configuring this at this point.

Reply:
OK, it is simple, true. Here you are.